tests/unittests/test_dftb_feeds.py

```diff
@@ -146,6 +146,120 @@ def test_scipyskfeed_batch(device, skf_file: str):
 # Note that gradient tests are not performed on the ScipySkFeed as it is not
 # backpropagatable due to its use of Scipy splines for interpolation.
 
+def test_skfeed_poly_single(device, skf_file: str):
+    """ScipySkFeed matrix single system operability tolerance test"""
+
+    if device.type == "cuda":
+        pytest.skip("Scipy splines do not support CUDA.")
```

Comment:
As this is the test for SKFeed, CUDA should also be supported.

Reply:
Tests performed on non-Scipy feeds will no longer be skipped when running on the GPU.